contrib/ci/generate_debian.py

```diff
@@ -81,7 +86,7 @@ def parse_control_dependencies(requested_type):
                                 suffix = "]"
                             dep = "%s%s%s%s" % (dep, prefix, exclusions[i].text, suffix)
                     deps.append(dep)
-    return deps
+        return deps, QUBES
```

Comment:
is this right? the indent level changed which made my spider senses tingle

Reply:
Fixed.